app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -104,8 +112,12 @@ def self.raw_connect(host, port, username, password, security_protocol, cert_sto
         :verify_ssl     => verify_ssl_mode(security_protocol),
         :ssl_cert_store => cert_store
```

Comment:
To fix the SSL error, please change this to:
``` ruby
:ssl_cert_store => if cert_store.kind_of?(String)
                     Endpoint.new(:certificate_authority => cert_store).ssl_cert_store
                   else
                     cert_store
                   end
```

Reply:
I think this should come as a separate PR, since it's fixing different issue.

Reply:
@israel-hdez fixed here https://github.com/ManageIQ/manageiq-providers-hawkular/pull/101